docs/source/contributor-guide/governance.md

```diff
@@ -0,0 +1,93 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# Governance
+
+The current PMC and committers are listed in the [Apache Phonebook].
+
+[apache phonebook]: https://projects.apache.org/committee.html?datafusion
+
+## Overview
+
+DataFusion is part of the [Apache Software Foundation] and is governed following
+the [Apache Way] and [project management guidelines], [independently of
+commercial interests].
+
+[apache software foundation]: https://www.apache.org/
+[apache way]: https://www.apache.org/theapacheway/
+[project management guidelines]: https://www.apache.org/foundation/how-it-works.html#management
+[independently of commercial interests]: https://community.apache.org/projectIndependence.html
+
+As much as practicable, we strive to make decisions by consensus, and anyone in
+the community is encouraged to propose ideas, start discussions, and contribute
+to the project.
+
+## Roles
+
+- **Contributors**: Anyone who contributes to the project, whether it be code,
+  documentation, testing, issue reports, code, or some other forms.
+
+- **Committers**: Contributors who have been granted write access to the
+- project's source code repository. Committers are responsible for reviewing
```

Comment:
I think these two list items are redundant.
![e23123e_capture](https://github.com/apache/datafusion/assets/26053282/f9859bdb-462a-4a90-98ff-3174808afba1)

Reply:
Excellent catch @jonahgao  -- fixed in e9a93ddb4